build.gradle

```diff
@@ -39,7 +44,9 @@ sourceSets {
 }
 
 dependencies {
-    spi(project(':spi'))
+    implementation(project(':spi'))
+    implementation('com.worksap.nlp:sudachi:0.7.4')
```

Comment:
This is required to load the sudachi jar in ES9. We can't use the previous dynamic loading anymore due to `entitlement-policy.yaml`